saleor/graphql/discount/mutations.py

```diff
@@ -332,7 +349,12 @@ def perform_mutation(cls, _root, info, **data):
         voucher = cls.get_node_or_error(
             info, data.get("id"), only_type=Voucher, field="voucher_id"
         )
-        cls.remove_catalogues_from_node(voucher, data.get("input"))
+        input = data.get("input")
+        cls.remove_catalogues_from_node(voucher, input)
```

Comment:
If `input` will be `None` then `remove_catalogues_from_node` will raise `'NoneType' object has no attribute 'get'`